### PR TITLE
chore: bump up Expo version in crna-kitchen-sink example

### PR DESCRIPTION
**Pull request by estevaolucas**
Issue: N/A

## What I did

Update expo requirement from `^24.0.2` to `^30.0.1` for `crna-kitchen-sink` example. Also remove [`crna`](https://github.com/react-community/create-react-native-app)'s `react-native-scripts` to use `expo`.

Motivation: I tried to use `crna-kitchen-sink`, but [I got stuck in "Starting packager"](https://github.com/react-community/create-react-native-app/issues/203).

PS: `whatwg-fetch` was added to fix this issue: ["Can't Find Variable Self"](https://stackoverflow.com/questions/52269560/react-native-expo-cant-find-variable-self)

## How to test

Is this testable with Jest or Chromatic screenshots? NO
Does this need a new example in the kitchen sink apps? NO
Does this need an update to the documentation? NO

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

File: examples-native/crna-kitchen-sink/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # CRNA Kitchen Sink
 
-This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app) and storybook using [storybook CLI](https://www.npmjs.com/package/@storybook/cli).
```
Comment (Gongreg) [inline examples-native/crna-kitchen-sink/README.md]: Maybe just change the sentence that it was bootstrapped with expo?

**Review by Gongreg — APPROVED** [review]

**Comment by Gongreg** [comment]
Thank you for updating the example :)

File: examples-native/crna-kitchen-sink/README.md (hunk 1 above)
Comment (elucaswork) [reply examples-native/crna-kitchen-sink/README.md]: done 👍

**Comment by Gongreg** [comment]
@elucaswork can you just confirm that the example is working? I mean try to do a fresh install and see that it still works.

**Comment by elucaswork** [comment]
@Gongreg I just double checked with a fresh install. It's working for iOS and Android 👍

**Pull request merged**